src/main/java/frc/robot/commands/vision/VisionAim.java

```diff
@@ -38,34 +39,28 @@ public VisionAim(Robot robot)
   @Override
   public void initialize()
   {
-    robot.vision.enableLight();
+   robot.vision.enableAimingLight();
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
     double turn = robot.vision.aimAtTarget(); //Get the turn speed from the camera
-
+  
     if (turn > 0) {
-      robot.drivetrain.moveMotors(0, TURN_SPEED);
+      robot.drivetrain.moveMotors(0, Drivetrain.AIM_TURN_SPEED);
     } else if (turn < 0) {
-      robot.drivetrain.moveMotors(0, -TURN_SPEED);
+      robot.drivetrain.moveMotors(0, -Drivetrain.AIM_TURN_SPEED);
```

Comment:
Not sure if it is easier for you to store `AIM_TURN_SPEED` in this vision aim function instead of storing constant in Drivetrain subsystem, so you can just change it here directly. But that can be your preference

Reply:
I moved it to Drivetrain because as I was creating different auto methods, I originally had multiple that needed `AIM_TURN_SPEED`, but now it's just this one. I'll keep it in Drivetrain in case we ever need to use it again somewhere else.